src/config.c

```diff
@@ -52,4 +57,69 @@ int ReadConfig(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     }
     RedisModule_Log(ctx, "verbose", "loaded default MAX_SAMPLE_PER_CHUNK policy: %lld \n", TSGlobalConfig.maxSamplesPerChunk);
     return TSDB_OK;
+}
+
+RTS_RedisVersion RTS_currVersion;
+
+RTS_RedisVersion RTS_supportedVersion = {
```

Comment:
why not name it `RTS_minSupportedVersion`?

Reply:
agree. changed in new revision.